apps/base/utils.py

```diff
@@ -103,7 +103,7 @@ def send_email(
     """
     try:
         sg = sendgrid.SendGridAPIClient(
-            apikey=os.environ.get("SENDGRID_API_KEY")
+            apikey=(os.environ.get("SENDGRID_API_KEY"))
```

Comment:
```suggestion
            apikey=os.environ.get("SENDGRID_API_KEY")
```
Okay, let's not make it a tuple. Lets keep it as is.

Reply:
@galipremsagar done